README.md

```diff
@@ -111,6 +111,14 @@ Got a list of default packages you want installed every time you install a new N
 set --universal nvm_default_packages yarn np
 ```
 
+## `$nvm_data`
+
+Set where nvm stores Node binaries and related data. Defaults to `$XDG_DATA_HOME/nvm` (~/.local/share/nvm) if unset.
+
+```fish
+set --global nvm_data ~/.nvm
```

Comment:
If set globally, it will not work.

```suggestion
set --universal nvm_data ~/.nvm
```

Reply:
What will not work exactly?

Reply:
You need to put it in your config.fish. Unless we intended this feature to work interactively, which I did not assume.

Reply:
It causes the following error:

```
nvm: Can't use Node "22", version must be installed first
```

Reply:
Of course I added that to config.fish, but it did not work unless I set it as universal.

Reply:
I see, thanks. I'll investigate then!

Reply:
It may have been shadowed by setting the variable in different ways.